website/docs/release_notes.md

```diff
@@ -5,6 +5,140 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.15](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.15)
+
+## M1 native launcher is here! (experimental)
+
+We are happy to announce that there is new dedicated launcher for M1 users.
```

Comment:
```suggestion
We are happy to announce that there is a new dedicated launcher for M1 users.
```

Reply:
Also, we might want to put a link to bug tickets regarding the stuff that doesn't yet work, like the `package` stuff